<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prostate</class>
 <widget class="qMRMLWidget" name="Prostate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>494</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Reconstruction</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>false</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="setSliceButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reconstruct Prostate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Overlay Zones</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MRI model:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="modelSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>MRI volume:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="MRVolumeSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registerButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Select zonal model:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="zoneSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="invertButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set Zones on TRUS</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ground Truth</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qMRMLNodeComboBox" name="GTSelector">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLModelNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Registered PZ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qMRMLNodeComboBox" name="RegZoneSelector">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLModelNode</string>
      </stringlist>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="metricsButton">
     <property name="text">
      <string>Get Metrics</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Prostate</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Prostate</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Prostate</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>modelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Prostate</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>zoneSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Prostate</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GTSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Prostate</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>RegZoneSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
